<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MN921Cfg</class>
 <widget class="QDialog" name="MN921Cfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>697</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLCDNumber" name="posobj">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="smallDecimalPoint">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>2</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Настройки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="netsetting">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Сеть</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="r168seting">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>R168</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="lineWidth">
          <number>30</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Канал 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Номер</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="channel_1_sipnum">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="inputMask">
              <string>9999</string>
             </property>
             <property name="text">
              <string>0000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="mb_ch1">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>МБ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cb_ch1">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ЦБ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tch_ch1">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ТЧ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ats_ch1">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>АТС</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="r168_ch1">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Р168100У2</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="r168100kb_ch1">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Р168100КБ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Громкость</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="gr_spin1">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::UpDownArrows</enum>
               </property>
               <property name="showGroupSeparator" stdset="0">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QDial" name="gr_dial1">
             <property name="maximum">
              <number>8</number>
             </property>
             <property name="tracking">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setabonent_1">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(186, 189, 182);</string>
           </property>
           <property name="text">
            <string>Абоненты</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setch_1">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(186, 189, 182);</string>
           </property>
           <property name="text">
            <string>Настройка</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="lineWidth">
          <number>30</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Канал 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Номер</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="channel_2_sipnum">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="inputMask">
              <string>9999</string>
             </property>
             <property name="text">
              <string>0000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="mb_ch2">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>МБ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cb_ch2">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ЦБ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tch_ch2">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>ТЧ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ats_ch2">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>АТС</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="r168_ch2">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Р168100У2</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="r168100kb_ch2">
             <property name="font">
              <font>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Р168100КБ</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Громкость</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="gr_spin2">
               <property name="font">
                <font>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="maximum">
                <number>8</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QDial" name="gr_dial2">
             <property name="maximum">
              <number>8</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setabonent_2">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(186, 189, 182);</string>
           </property>
           <property name="text">
            <string>Абоненты</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setch_2">
           <property name="font">
            <font>
             <pointsize>26</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(186, 189, 182);</string>
           </property>
           <property name="text">
            <string>Настройка</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="loadfromfile">
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
         <property name="text">
          <string>Загрузить из файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savetofile">
         <property name="font">
          <font>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(186, 189, 182);</string>
         </property>
         <property name="text">
          <string>Сохранить в файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savetodevice">
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(114, 159, 207);</string>
       </property>
       <property name="text">
        <string>Сохранить на устройство</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
